.circleci/config.yml

```diff
@@ -2,10 +2,12 @@ version: 2
 jobs:
   build:
     machine:
-      image: circleci/classic:201708-01
+      image: ubuntu-2004:202101-01
```

Comment:
why did we moved to an ubuntu image from circleci?

Reply:
It is no longer working so had to update the image.